client/state/selectors/is-editor-loaded.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * Internal dependencies
+ */
+
+import { isEditorLoaded } from 'state/ui/editor/selectors';
+
+export default state => {
+	return isEditorLoaded( state );
+};
```

Comment:
Can we have just one unique definition of the `isEditorLoaded` selector? This is a module that just re-exports a selector that could be as well imported directly from the other location.

Reply:
Removed redundant module export in 3906461. 👍